pkg/csi/service/wcpguest/controller.go

```diff
@@ -1345,11 +1355,31 @@ func (c *controller) ListVolumes(ctx context.Context, req *csi.ListVolumesReques
 
 func (c *controller) GetCapacity(ctx context.Context, req *csi.GetCapacityRequest) (
 	*csi.GetCapacityResponse, error) {
-
 	ctx = logger.NewContextWithLogger(ctx)
 	log := logger.GetLogger(ctx)
 	log.Infof("GetCapacity: called with args %+v", *req)
-	return nil, status.Error(codes.Unimplemented, "")
+
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.WorkloadDomainIsolationFSS) {
+		var totalcapacity, maxvolumesize int64
+		zonesMap := commonco.ContainerOrchestratorUtility.GetZonesForNamespace(c.supervisorNamespace)
+		if req.AccessibleTopology != nil {
+			segments := req.AccessibleTopology.GetSegments()
+			if _, exists := zonesMap[segments["topology.csi.vmware.com/k8s-zone"]]; !exists {
```

Comment:
incorrect key, need change key from "topology.csi.vmware.com/k8s-zone" to "topology.kubernetes.io/zone". Without this change, GetCapacity() always returns zero capacity